includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -732,11 +732,24 @@ public function payment_fields() {
 		try {
 			$display_tokenization = $this->supports( 'tokenization' ) && ( is_checkout() || is_add_payment_method_page() );
 
-			add_action( 'wp_footer', [ $this, 'enqueue_payment_scripts' ] );
+			wp_add_inline_script(
```

Comment:
This is not a solution... the same issues appear with WP 5.9 and Twenty Twenty-Two. `wcpay_config` doesn't appear anywhere within the code, unless it's wrapped within a `wp_footer()` call.

To reproduce:
1. Check out this PR
2. Add a product to the cart
3. Open the non-block checkout page, and see that payment fields are not working again.